link-parent/link-core/src/main/java/org/interledger/link/exceptions/LinkException.java

```diff
@@ -85,4 +85,17 @@ public LinkException(
   public LinkId getLinkId() {
     return linkId;
   }
+
+  /**
+   * Overridden to emit the Link Id.
+   *
+   * @return A {@link String} representation of this Exception.
+   */
+  @Override
+  public String toString() {
+    String s = getClass().getName();
+    String message = getLocalizedMessage() + " LinkId=" + getLinkId();
+    return (message != null) ? (s + ": " + message) : s;
```

Comment:
I can't see now message would ever be null

Reply:
Fixed here: https://github.com/hyperledger/quilt/commit/9e83dd74ad8526f0e28bea2e6a04ac880c8c5443